scripts/package.json

```diff
@@ -18,7 +18,6 @@
     "@microsoft/load-themed-styles": "^1.7.13",
     "@microsoft/loader-load-themed-styles": "^1.6.0",
     "@octokit/rest": "^16.28.2",
-    "@uifabric/codepen-loader": "^7.0.8",
```

Comment:
I don't think we need the jscodeshift, babylon deps anymore after taking this out...